Packs/AccentureCTI_Feed/Integrations/ACTIIndicatorFeed/ACTIIndicatorFeed.py

```diff
@@ -0,0 +1,146 @@
+from typing import Dict, Union
+from CommonServerPython import *
+from JSONFeedApiModule import *  # noqa: E402
+
+
+def custom_build_iterator(client: Client, feed: Dict, limit, **kwargs) -> List:
+    """
+    Implement the http_request with API that works with pagination and filtering. Uses the integration context to
+    save last fetch time to each indicator type
+    Args:
+        client: Client manage all http requests
+        feed: dictionary holds all data needed to the specific service (Services- IP, Domain, URL)
+        limit: maximum number of indicators to fetch
+
+    Returns:
+        list of indicators returned from api. Each indicator is represented in dictionary
+    """
+    fetch_time = demisto.params().get('fetch_time', '14 days')
```

Comment:
Please move this line to `main()` (using `params` variable rather than `demisto.params()`) and pass the `fetch_time` as an argument to the function.

Reply:
Has a question to ask. `custom_build_iterator` is the function where you wanted me to pass `fetch_time`. But seems like it is getting called in `create_fetch_configuration` function and there `custom_build_iterator` is being called with no arguments getting passed to that. 

And while debugging  I got to know that somehow `custom_build_iterator` is getting executed in `main()` under `feed_main` function.

So was a bit confused on how to pass `fetch_time` to `custom_build_iterator` function.

Reply:
@satyakidroid I see. Never mind my comment then.